pkg/actuators/machine/actuator_test.go

```diff
@@ -1,1490 +0,0 @@
-package machine
```

Comment:
Is this being moved somewhere else? otherwise this would just drop unit coverage

Reply:
Yes, tests were partially distributed to machine scope and I'm planning to create a follow-up for bringing back tests for events we fire and etc. We also need the same tests for vSphere, its actuator is missing them too.

Reply:
Before:
```
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machine	9.638s	coverage: 85.8% of statements
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machineset	10.799s	coverage: 85.7% of statements
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/apis	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsprovider	[no test files]
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsprovider/v1beta1	0.252s	coverage: 37.4% of statements
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client/fake	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client/mock	[no test files]
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/termination	13.309s	coverage: 79.6% of statements
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/version	[no test files]
```

After
```
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machine	7.199s	coverage: 65.4% of statements
ok  	sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machineset	8.615s	coverage: 88.1% of statements
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/apis	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsprovider	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsprovider/v1beta1	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client/fake	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/client/mock	[no test files]
?   	sigs.k8s.io/cluster-api-provider-aws/pkg/version	[no test files]
```

I'd like to help to move this forward, can we make sure we increase the 65.4% before merging?